omegaconf/basecontainer.py

```diff
@@ -228,7 +228,30 @@ def pretty(self, resolve: bool = False, sort_keys: bool = False) -> str:
 
         def str_representer(dumper: yaml.Dumper, data: str) -> yaml.ScalarNode:
             def is_bool(data: str) -> bool:
-                return data in ["True", "False"]
+                return data in [
```

Comment:
Looks like this is from https://yaml.org/type/bool.html ?

1. Keep the same order.
2. Add a link to it in a comment.

Reply:
move this logic somewhere else. _utils?